app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,35 @@
+class AdminsController < ApplicationController
+  before_action :check_admin
+
+  def index
+    @users = User.all
+    @usertypes = Usertype.all
+  end
+
+  def show
+    user = User.find(params[:id])
+    puts user.inspect
+    @productions = user.designed_productions if user.user_type == "Designer"
+    @productions = user.lead_productions if user.user_type == "Lead"
```

Comment:
you should raise an exception in the case where the `user_type` is nil. right now `@productions` will be `nil` which could break your views